README.md

```
@@ -264,6 +264,22 @@ Control YouTube player rendering method and configure source URLs for compatibil
 />
 ```
 
+**Custom Player Page**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)